### ISSUE TITLE
Ship-maneuver-roll uses weapon-template

### ISSUE THREAD

**Issue by kagedansa** [kbender84/token-action-hud-coriolis#2]
![image](https://github.com/kbender84/token-action-hud-coriolis/assets/110729103/6e537d60-1a0d-461a-a211-329dc2f6718c)

**Comment by kbender84** [comment]
Hi, yes, this is by design... Unfortunately Coriolis core system has just few templates, and basic rolls (attribute and skill) do not take Bonus into account.
I wanted to be clear that there is ship Bonus involved, separately from roll modifier so I have used weapon template (with all the extra fields it brings with it). Alternative would be to add ship manoeuvrability bonus to the roll directly, but then people would ask "why the modifier is +2, when I selected +1". 
Not sure how can I handle it at this moment without hacking Coriolis core module again (and I really don't want to go this path again :) )
